tests/config_validator/test_validate.py

```diff
@@ -1,111 +1,114 @@
+# pylint: disable=bad-continuation
```

Comment:
This file has already been reviewed in https://github.com/ChauffeurPrive/nestor-api/pull/41/files

Reply:
I meant https://github.com/ChauffeurPrive/nestor-api/pull/40